modules/@apostrophecms/modal/ui/apos/components/TheAposModals.vue

```diff
@@ -17,6 +17,10 @@
 <script>
 import cuid from 'cuid';
 import AposThemeMixin from 'Modules/@apostrophecms/ui/mixins/AposThemeMixin';
+import { useModalStore } from 'Modules/@apostrophecms/ui/stores/modal';
```

Comment:
How much value is a new dependency on Pinia bringing? `apos.modal.modals` has been visible in the code as a property other modules might try to look at for a while now - can we maintain that interface to reduce backwards compatibility issues?

Reply:
`apos.modal.modals` comes from the backend and is static data so we have no reason to put it does not move.

Reply:
Ohh right, never mind.